examples/blender/blender.py

```diff
@@ -20,13 +9,23 @@
     print_env_info,
     run_golem_example,
 )
+from datetime import datetime, timedelta
+import pathlib
+import sys
+
+from yapapi import Golem, Task, WorkContext
+from yapapi.payload import vm
+from yapapi.rest.activity import BatchTimeoutError
+
+examples_dir = pathlib.Path(__file__).resolve().parent.parent
+sys.path.append(str(examples_dir))
 
 
 async def main(
     subnet_tag, min_cpu_threads, payment_driver=None, payment_network=None, show_usage=False
 ):
     package = await vm.repo(
-        image_hash="9a3b5d67b0b27746283cb5f287c13eab1beaa12d92a9f536b747c7ae",
+        image="9a3b5d67b0b27746283cb5f287c13eab1beaa12d92a9f536b747c7ae",
```

Comment:
any such breaking changes should follow a proper deprecation process...

BUT, having said that, I don't think that we want to have it implemented that way - rather I _think_ we'd like to explicitly differentiate tags from hashes

Reply:
I agree on the deprecation process - i'm just unsure how it's usually done, so please do suggest what to do.

I just think there will be less struggle for requestors if they can just define a single parameter to the package, and insert either a hash or tagged image and then yapapi automatically handles it for them. If we decide to split it up anyway, wouldn't it be smart to include some kind of check to see if they've accidentally used a hash on the tag parameter and vice versa?